app/controllers/preset.controller.js

```diff
@@ -0,0 +1,38 @@
+const { presets } = require("../models");
+const db = require("../models");
+const Presets = db.presets;
+const Op = db.Sequelize.Op;
+
+exports.get = (req, res) => {
+
+    condition = req.params.id ? {ID: req.params.id} : null;
+
+    Presets.findAll({ where: condition })
+    .then( data => {
+        res.send(data);
+    })
+    .catch(err => {
+        res.status(500).send({
+          message:
+            err.message || "Some error occurred while retrieving tutorials."
+        });
+      });
+}
+
+exports.getFromUser = (req, res) => {
+
+    condition = req.params.id ? {user_id: req.params.id} : null;
```

Comment:
This line is not necessary, because for this function to work, the where clause is not optional. You also need to add a line to check if the parameter id exists, if it doesn't, cancel the request (send me a msg if you need help or have trouble understanding this comment)